src/Console/IdeHelperCommand.php

```diff
@@ -108,6 +111,14 @@ protected function buildSchemaString(array $directiveClasses): string
                 .$definition."\n";
         }
 
+        // append programmatically registered types
+        $typeRegistry = resolve(TypeRegistry::class);
+        $typeRegistry->setDocumentAST(new DocumentAST);
+        $programmaticallyRegisteredTypes = $typeRegistry->possibleTypes();
```

Comment:
This actually loads SDL defined types, too.

Reply:
I don't think so. I pass an empty `DocumentAST` instance, with an empty schema. When `possibleTypes()` gets executed, `$this->documentAST->types` in line 182 in `TypeRegistry.php` is an empty array, so the function simply returns `$this->types`, and there are only types, that were registered with `$typeRegistry->register()`

Reply:
I tried the command in my live project - only programmatically registered types are considered and printed into output file

Reply:
Nice! I appreciate the creativity, but found that adding an empty `DocumentAST` is a bit of a hacky workaround. I added `resolvedTypes` to `TypeRegistry` to make this more straightforward.